cms/djangoapps/contentstore/management/commands/tests/test_reindex_library.py

```diff
@@ -130,7 +130,10 @@ def test_given_all_key_prompts_and_reindexes_all_libraries(self):
 
                 patched_yes_no.assert_called_once_with(ReindexCommand.CONFIRMATION_PROMPT, default='no')
                 expected_calls = self._build_calls(self.first_lib, self.second_lib)
-                self.assertEqual(patched_index.mock_calls, expected_calls)
+                # Call order may depend on python environment, but all should be there.
+                calls = sorted(patched_index.mock_calls, key=lambda x: x[1][1].library)
+                expected_calls = sorted(expected_calls, key=lambda x: x[1][1].library)
+                self.assertEqual(calls, expected_calls)
```

Comment:
+1 on this.  I meant to mention it to you directly after the meeting but forgot about it.  For hashable objects, `assertItemsEqual()` will essentially compare `set(actual)` and `set(expected)` with taking into account multiplicities if elements are repeated, but it will even work for unhashable objects, falling back to an O(n²) algorithm.  So for long lists of unhashable objects, you might want to find a different solution.